guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -219,6 +219,161 @@ Create the host from an image with the `hammer host create` command and include
 
 For more information about additional host creation parameters for this compute resource, see xref:CLI_Params[].
 
+[[Provisioning_Virtual_Machines_in_VMware_vSphere-Provisioning_with_cloudinit_and_userdata_templates]]
+=== Using the VMware vSphere cloud-init and userdata Templates for Provisioning
+
+You can use VMware with the `Cloud-init` and `userdata` templates to insert user data into the new virtual machine, to make further VMware customization, and to enable the VMware-hosted virtual machine to call back to {Project} at intervals.
+
+You can use the same procedures to set up a VMware compute resource within {Project}, with a few modifications to the work flow.
+
+.VMware cloud-init Provisioning Overview
+
+When you set up the compute resource and images for VMware provisioning in {Project}, the following sequence of provisioning events occur:
+
+* The user provisions one or more virtual machines using the {Project} web UI, API, or hammer
+* {Project} calls the VMware vCenter to clone the virtual machine template
+* {Project} `userdata` provisioning template adds customized identity information
+* When provisioning completes, the {Project} `cloud-init` provisioning template instructs the virtual machine to call back to {SmartProxy} when `cloud-init` runs
+* VMware vCenter clones the template to the virtual machine
+* VMware vCenter applies customization for the virtual machine's identity, including the host name, IP, and DNS
+* The virtual machine builds, `cloud-init` is invoked and pings {Project} on port `80`, which then redirects to `443`
+
+.Port and Firewall Requirements
+
+Because of the `cloud-init` service, the virtual machine always calls back to {Project} even if you register the virtual machine to {SmartProxy}. Ensure that you configure port and firewall settings to open any necessary connections.
+
+ifeval::["{build}" == "satellite"]
+For more information about port and firewall requirements, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.4/html/installing_satellite_server_from_a_connected_network/preparing_your_environment_for_installation#ports_prerequisites[Port and Firewall Requirements] in the _Installation Guide_.
+endif::[]
```

Comment:
I think we can keep most of the links even upstream, the ultimate goal is to have ALL guides converted. We should also fix all links base URLs but let's not do it here - I will automate this using some script. Noted: https://github.com/theforeman/foreman-documentation/issues/20

Reply:
Will do. Thank you